devicetypes/smartthings/motion-detector.src/motion-detector.groovy

```diff
@@ -34,14 +35,34 @@ metadata {
 				attributeState("active", label:'motion', icon:"st.motion.motion.active", backgroundColor:"#00A0DC")
 				attributeState("inactive", label:'no motion', icon:"st.motion.motion.inactive", backgroundColor:"#CCCCCC")
 			}
- 		}
+		}
 		main "motion"
 		details "motion"
 	}
 }
 
+def installed() {
+	initialize()
+}
+
+def updated() {
+	initialize()
+}
+
+def initialize() {
+	if (isUntracked()) {
+		log.debug "device untracked"
+		sendEvent(name: "DeviceWatch-Enroll", value: JsonOutput.toJson([protocol: "zigbee", scheme:"untracked"]), displayed: false)
+	} else {
+		// Device-Watch simply pings if no device events received for 12min(checkInterval)
+		log.debug "device tracked"
+		sendEvent(name: "checkInterval", value: 10 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
```

Comment:
does this device send us a report every 6 minutes?

Reply:
Device sends a report every 5 minutes when there is no movement.